app/components/Home.tsx

```diff
@@ -35,22 +39,29 @@ const MainBrowserWindow: React.FunctionComponent<Props> = (props: Props) => {
     updateTabs(modifiedTabs);
     setTabIndex(modifiedTabs.length - 1);
     updateUrl('');
-    setModal(modal);
+    updateTitle('');
+    setModal(false);
   }
 
   const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => updateTitle(event.target.value)
   const onUrlChange = (event: React.ChangeEvent<HTMLInputElement>) => updateUrl(event.target.value)
 
   const onRemovingTab = (index: number) => {
-    const tabItem = tabs.filter((_,tabIndex) => tabIndex !== index)
-    updateTabs(tabItem);
+    const tabItems: BrowserTab = tabs.filter((_,tabIndex: number) => tabIndex !== index)
+    updateTabs(tabItems);
   }
   return (
     <>
       {
         tabs.map(
           (tab: BrowserTab, index: number) =>  {
-            return <button className={`urlview ${mainTabIndex === index ? 'active' : ''}`} onClick={() => setTabIndex(index)} key={index}><img src="https://recraftrelic.com/images/Recraft_relic_web_logo_icon.png" />{tab.title}<Fonticon className="fa fa-times" onClick={() => onRemovingTab(index)} key={index}></Fonticon></button>
+            return <TabsManager 
+              className={`urlview ${mainTabIndex === index ? 'active' : ''}`} 
```

Comment:
Can't we put this className logic inside the `TabsManager` ?  just pass a new prop `isActive` and handle the `ternary` inside